druid/src/command.rs

```diff
@@ -54,9 +61,8 @@ pub struct Selector(&'static str);
 /// [`Selector`]: struct.Selector.html
 #[derive(Debug, Clone)]
 pub struct Command {
-    /// The command's `Selector`.
-    pub selector: Selector,
-    object: Option<Arc<dyn Any>>,
+    selector: SelectorSymbol,
```

Comment:
Can we rename this `selector` to e.g. `selector_symbol`? I think using `selector` to mean two different types can be confusing.

Reply:
Sounds good. I went with just `symbol` as its meaning is unique in the `Command` context.